module/dkg/controller_factory.go

```diff
@@ -49,30 +49,24 @@ func (f *ControllerFactory) Create(
 	participants flow.IdentityList,
 	seed []byte) (module.DKGController, error) {
 
-	myIndex := -1
-	for i, id := range participants.NodeIDs() {
-		if id == f.me.NodeID() {
-			myIndex = i
-			break
-		}
-	}
-	if myIndex < 0 {
-		return nil, fmt.Errorf("node does not belong to dkg committee")
+	myIndex, ok := participants.GetIndex(f.me.NodeID())
+	if !ok {
+		return nil, fmt.Errorf("failed to create controller factory, node %s is not part of DKG committee", f.me.NodeID().String())
```

Comment:
Do we need to error in this case? Shouldn't it be a "normal" case that a consensus node is not part of the beacon committee? 
cc @jordanschalm

Reply:
It is an expected case that's true. A sentinel would be better so the caller can distinguish between "expected, but we can't initialize the DKG" and "unexpected error"